apps/server/src/routes/apiRouter.ts

```diff
@@ -14,9 +14,19 @@ router.get('/', (_req, res) => {
 
 // any GET request in /api is sent to the integration controller
 router.get('/*', (req, res) => {
-  const action = req.path.substring(1);
+  let action = req.path.substring(1);
+  const actionArray = action.split('/');
+  
```

Comment:
todo: remember to run the formatter in your IDE. It is easy to get silly merge conflicts